TS6133 in settings views: unused `v-for` variables

TypeScript compilation fails with TS6133 errors in several settings views because variables declared in `v-for` expressions are never read. The build treats these as hard errors instead of non-blocking warnings.

## Affected files

- `src/views/settings/EnergyMonitorSettingsView.vue`
- `src/views/settings/ForecastProvidersSettingsView.vue`
- `src/views/settings/MinerControllersSettingsView.vue`

## Error details

```text
src/views/settings/EnergyMonitorSettingsView.vue:109:19 - error TS6133: 'energyMonitor' is declared but its value is never read.

109           v-for="(energyMonitor, i) in energyMonitorStore.energyMonitors"
                      ~~~~~~~~~~~~~

src/views/settings/ForecastProvidersSettingsView.vue:105:19 - error TS6133: 'forecastProvider' is declared but its value is never read.

105           v-for="(forecastProvider, i) in forecastProviderStore.forecastProviders"
                      ~~~~~~~~~~~~~~~~

src/views/settings/MinerControllersSettingsView.vue:105:19 - error TS6133: 'minerController' is declared but its value is never read.

105           v-for="(minerController, i) in minerControllerStore.minerControllers"
                      ~~~~~~~~~~~~~~~

Found 3 errors.
```

## Impact

- `npm run build` fails due to TS6133, even though the underlying issue is only unused `v-for` variables in templates.
- The error currently prevents producing a successful production build for the frontend.

--- a/src/views/settings/EnergyMonitorSettingsView.vue
+++ b/src/views/settings/EnergyMonitorSettingsView.vue
@@ -105,11 +105,9 @@ const formatAdapterType = (type: string) => {
         </tr>
       </thead>
       <tbody>
-        <template
-          v-for="(energyMonitor, i) in energyMonitorStore.energyMonitors"
-          :key="energyMonitor.id"
-        >
+        <template>
           <EnergyMonitorRow
+            v-for="(energyMonitor, i) in energyMonitorStore.energyMonitors" :key="energyMonitor.id"
             v-model="energyMonitorStore.energyMonitors[i]"
             @edit="handleEdit"
             @delete="handleDelete"

--- a/src/views/settings/ForecastProvidersSettingsView.vue
+++ b/src/views/settings/ForecastProvidersSettingsView.vue
@@ -101,11 +101,9 @@ const formatAdapterType = (type: string) => {
         </tr>
       </thead>
       <tbody>
-        <template
-          v-for="(forecastProvider, i) in forecastProviderStore.forecastProviders"
-          :key="forecastProvider.id"
-        >
+        <template>
           <ForecastProviderRow
+            v-for="(forecastProvider, i) in forecastProviderStore.forecastProviders" :key="forecastProvider.id"
             v-model="forecastProviderStore.forecastProviders[i]"
             @edit="handleEdit"
             @delete="handleDelete"

--- a/src/views/settings/MinerControllersSettingsView.vue
+++ b/src/views/settings/MinerControllersSettingsView.vue
@@ -101,11 +101,9 @@ const formatAdapterType = (type: string) => {
         </tr>
       </thead>
       <tbody>
-        <template
-          v-for="(minerController, i) in minerControllerStore.minerControllers"
-          :key="minerController.id"
-        >
+        <template>
           <MinerControllerRow
+            v-for="(minerController, i) in minerControllerStore.minerControllers" :key="minerController.id"
             v-model="minerControllerStore.minerControllers[i]"
             @edit="handleEdit"
             @delete="handleDelete"

